src/CMakeLists.txt

```diff
@@ -106,7 +106,6 @@ if(DLAF_WITH_PRECOMPILED_HEADERS)
     $<$<BOOL:${DLAF_WITH_CUDA}>:<pika/cuda.hpp$<ANGLE-R>>
     <pika/execution.hpp>
     <pika/future.hpp>
```

Comment:
It's not actually used included/used anywhere at the moment because of the previously mentioned performance difference. The `transformMPI` implementation in `transform_mpi.h` is self-contained and all there is to it at the moment.